src/components/implementation/no_interface/cbboot/quarantine.c

```diff
@@ -89,7 +90,7 @@ void* quarantine_translate_addr(spdid_t spdid, vaddr_t addr)
 		printd("cos_vect_lookup(%d) in spd_sect_cbufs failed\n", spdid);
 	if (unlikely(!(h = cos_vect_lookup(&spd_sect_cbufs_header, spdid))))
```

Comment:
My eyes are blurring over, so I'm going to go through the rest, but I'll focus on a lot of these latter files more in subsequent reviews.